src/main/java/picard/sam/markduplicates/util/OpticalDuplicateFinder.java

```diff
@@ -50,7 +50,7 @@
 
```

Comment:
This one doesn't show up in picardDoc. The javaDoc could be formatted better, but it's OK now.

Reply:
I think this is a utility called upon by other tools and is not a standalone tool itself.